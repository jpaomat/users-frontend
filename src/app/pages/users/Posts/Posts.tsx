import { useContext } from 'react';
import { usePosts } from '../../../hooks';
import { UserTemplate } from '../../../templates';
import envJson from '../../../config/ENV.json';
import { UserContext } from '../../../context';

export const Posts = () => {
	const usersClient = envJson.usersClient;
	const { usersState } = useContext(UserContext);
	const { postsData, isLoadingData } = usePosts(usersClient.post, usersState.numberPosts);

	return <UserTemplate dataCardList={postsData} isLoadingData={isLoadingData} />;
};
