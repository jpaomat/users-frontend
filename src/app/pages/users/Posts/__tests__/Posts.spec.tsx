import { render } from '@testing-library/react';
import { Posts } from '../Posts';

describe('<Albums /> component', () => {
    it('should render component', () => {
		const {container} = render(<Posts />);
		expect(container).toMatchSnapshot();
	});
});
