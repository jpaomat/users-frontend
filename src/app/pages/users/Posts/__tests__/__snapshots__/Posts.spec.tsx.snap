// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Albums /> component should render component 1`] = `
<div>
  <div>
    Albums
  </div>
</div>
`;
